docs/logging.md

```diff
@@ -0,0 +1,137 @@
+# Feature Logging
+
+To extract feature scores of a feature set or a model the `ltr_log` search extension.
+The `ltr_log` extension is able to capture feature values of a `sltr` query use inside the main search query when
+`named` or used inside a `rescore_query`.
+
+Example to output the feature scores for the feature set `my_feature_set` and for the model `my_model`:
+
+```json
+{
+  "query" : {
+    "bool" : {
+      "must" : {
+        "sltr" : {
+          "params" : {
+            "query_string" : "a search query"
+          },
+          "featureset": "my_feature_set",
+          "_name": "logged_featureset"
+        },
+        "filter": {
+          "ids" : {
+            "type" : "my_type",
+            "values" : ["1", "4"]
+          }
+        }
+      }
+    }
+  },
+  "rescore": [
+    {
+      "window_size": 3,
+      "query": {
+        "rescore_query": {
+          "sltr": {
+            "params" : {
+              "query_string" : "a search query"
+            },
+            "model": "my_model"
+          }
+        }
+      }
+    }
+  ],
+  "size": 3,
+  "ext": {
+    "ltr_log" : {
+      "log_specs": [
+        {
+          "name": "log_entry1",
+          "named_query": "logged_featureset",
```

Comment:
Turns out this is me not reading the documentation well enough. Elasticsearch doesn't parse names out of rescore queries, so rescore *must* use rescore_index and sltr in the main query *must* use named_query.